### ISSUE TITLE
can't configure php to get docblock

### ISSUE THREAD

**Issue by zordsdavini** [kkoomen/vim-doge#71]
Installed doge via Plug. Tool itself works - tried with python but I can't get it working for php. Maybe there should be some additional installs? Thought about PhpDocumentor2 but DoGe didn't start working

**Comment by kkoomen** [comment]
Hi. Every language, besides C or C++, will work out-of-the-box without the use of third party. Only C and C++ require third part libs. You can simple do this:

```php
function greet(string $name) {
  echo "Hello " . $name;
}
```

Keep your cursor at line 1 here and press `<Leader>d`. It should give you something like this:

```
/**
 * [TODO:description]
 *
 * @param string $name [TODO:description]
 * @return [TODO:type] [TODO:description]
 */
function greet(string $name) {
  echo "Hello " . $name;
}
```

**Comment by kkoomen** [comment]
If you can't get it to work, then I suggest you paste your code here you try to generate documentation for. Maybe you have found a scenario that doesn't generate properly.

**Comment by zordsdavini** [comment]
```    public function koja(string $ggg): array
    {
        throw new \Exception('sfdsfdsds');
    }
```
I removed return type and it started working. Is there possibilities to show return type. And maybe there is example how template can be adjusted: I don't need all helper texts and params should be tabbed, space before return. And as I understand Exceptions are not shown, too

**Comment by zordsdavini** [comment]
the result what I want is
```php
    /**
     * @param string $ggg
     * @param array  $a
     *
     * @return array
     * @throws \Exception
     */
    public function koja(string $ggg, array $a): array
    {
        throw new \Exception('sfdsfdsds');
    }
```

**Comment by kkoomen** [comment]
Ah, yeah, since a recent php 7 version it's possible to add return types, which hasn't been supported. I'll add support for this after #65 has been merged.

**Comment by kkoomen** [comment]
This feature has been merged and released in [v2.1.0](https://github.com/kkoomen/vim-doge/tree/v2.1.0).

Feel free to submit any new issues if you experience any unwanted behavior in the future. Thanks for your contribution.
